api/README.md

```diff
@@ -1,8 +1,17 @@
-The image `aixigo/prevant` provides the REST-API in order to deploy containers and to compose them into reviewable application.
+In this section, you can find all the configuration guidelines.
```

Comment:
Please, remove because the structure should tell you what is contained in this doc.

```suggestion
```